doc/lua_api.txt

```diff
@@ -845,7 +831,7 @@ the global `minetest.registered_*` tables.
 
 * `minetest.unregister_biome(name)`
     * Unregisters the biome name from engine, and deletes the entry with key
-    * `name` from `minetest.registered_biome`
+      `name` from `minetest.registered_biome`
```

Comment:
(Same as above)

Reply:
This change is fine, the 2nd asterisk shouldn't be there.